src/main/java/core/basesyntax/servise/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.servise.impl;
+
+import core.basesyntax.model.OperationsList;
+import core.basesyntax.servise.FileReader;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+
+public class FileReaderImpl implements FileReader {
+    private static final int HEAD_OF_INPUT = 0;
+
+    @Override
+    public void read(File file) {
```

Comment:
void methods are hard to test
let's return `List<String>` from this method and then pass it to another service